src/main/java/org/elasticsearch/action/trainnaivebayes/TrainNaiveBayesResponse.java

```diff
@@ -24,66 +24,44 @@
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.xcontent.ToXContent;
 import org.elasticsearch.common.xcontent.XContentBuilder;
-import org.elasticsearch.common.xcontent.XContentBuilderString;
 
 import java.io.IOException;
 
 public class TrainNaiveBayesResponse extends ActionResponse implements ToXContent {
 
-    public String getIndex() {
-        return index;
-    }
-
-    public String getType() {
```

Comment:
Should we really not return type and index? I found it convenient when I wanted to look at the script after it was trained. I always forget whether the index is .scripts or .script and also was never sure which was the name of the language.